Dockerfile

```
@@ -1,6 +1,28 @@
-FROM rust:1.85 AS builder
+FROM lukemathwalker/cargo-chef:latest-rust-1.85.1-slim AS chef
 
 WORKDIR /usr/src/restations-builder
+
+FROM chef AS planner
+
+COPY ./Cargo.lock .
+COPY ./Cargo.toml .
+COPY ./cli ./cli
+COPY ./config ./config
+COPY ./macros ./macros
+COPY ./rust-toolchain.toml .
+COPY ./web ./web
+COPY ./db ./db
+COPY ./stations.sqlite.db ./stations.sqlite.db
```

```suggestion
COPY . .
```

No need to make it more complex than this, it won't affect dependency caching (since that depends exclusively on the has of `recipe.json`).